src/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -179,7 +179,6 @@ public String remoteShutdown(final String hashedPassword, final String salt) {
     if (encryptedPassword.equals(hashedPassword)) {
       (new Thread(() -> {
         System.out.println("Remote Shutdown Initiated.");
-        ThreadUtil.sleep(1000);
```

Comment:
@DanVanAtta What is the point of a new Thread if you are just going to immediately call System.exit(0)? My guess is this was to give something 1 second to complete. No idea if that makes any real sense but if we are gonna remove the sleep then shouldn't we remove the new thread as well?